### ISSUE TITLE
About the loss fucntion of ccgan

### ISSUE THREAD

**Issue by USTClj** [eriklindernoren/Keras-GAN#4]
Hi~ Grate for your awesome work! I have a doubt below:
in https://github.com/eriklindernoren/Keras-GAN/blob/master/ccgan/ccgan.py#56
'self.combined.compile(loss=['mse', 'binary_crossentropy'],'
The paper didn't say that the mse loss of generated in-painted image and real images is used?

**Comment by USTClj** [comment]
I have run this code but the result seems incorrect.

**Comment by eriklindernoren** [comment]
Yeah, I wrote about this in the README. 

"In this implementation I have also decided to combine the adversarial loss with an l2 loss which measures the generated image's similarity to the original images. These losses are weighted similar to the approach described by Pathak et al. (2016)
in their paper Context Encoders: Feature Learning by Inpainting."

Adding the l2 loss helps the model converge more quickly. It's possible to get convergence without it too of course, but it takes way longer time.
